apps/console/src/features/email-templates/utils/email-template-utils.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EmailTemplateManagementConstants } from "../constants";
+
+/**
+ * Utility class for Email template.
+ */
+export class EmailTemplateUtils {
+
+    /**
+     * Private constructor to avoid object instantiation from outside
+     * the class.
+     *
+     */
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    private constructor() { }
+
+    /**
+     * Filter governance categories of a connector for a sub organization.
+     * @param emailTemplateData - List of email template data that recieved to the console.
+     * 
+     * @returns Filtered templates as a list.
+     */
+    public static filterEmailTemplateTypesForOrganization
+    (emailTemplateData: any[]): any[] {
+
+        for (let i = emailTemplateData.length - 1; i >= 0; i--) {
+            if (!EmailTemplateManagementConstants.EMAIL_TEMPLATE_TYPES_FOR_ORGS.includes(emailTemplateData[i].id)) {
+                emailTemplateData.splice(i, 1);
+            }
+        }
+
+        return emailTemplateData;
```

Comment:
Can we use the filter API to make this more readable? 
https://www.w3schools.com/jsref/jsref_filter.asp